extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -129,10 +133,20 @@ export class SubscriptionService {
         /** @type {!PageConfig} */
         this.pageConfig_ = promiseValues[1];
       });
+      this.maybeDecryptDocument_();
     }
     return this.initialized_;
   }
 
+  /**
+   * @private
+   * @return {!Promise}
+   */
+  maybeDecryptDocument_() {
+    return this.cryptoHandler_.tryToDecryptDocument(
+        this.getEncryptedDocumentKey('google'));
```

Comment:
This should be `google.com` and also, why here? I see you're already calling it in the SwG platform.